src/nginx_manager.py

```diff
@@ -154,8 +162,12 @@ def _create_server_config(host: str, configuration: HostConfig) -> None:
     """
     logger.info("Creating the nginx site configuration file for hosts %s", host)
     try:
-        nginx_config = nginx.Conf()
+        proxy_cache_path = NGINX_PROXY_CACHE_DIR_PATH / host
+        nginx_config = nginx.Conf(
+            nginx.Key("proxy_cache_path", f"{proxy_cache_path} keys_zone={host}:10m")
```

Comment:
Is `10m` a good cache size ? What was the size previously ?

Reply:
`10m` is the default size for the current charm.

Reply:
The `:10m` specifies the size of all active keys and metadata of the cache. Not the cache itself.